src/Security/SecurityControllerBuilder.php

```diff
@@ -22,11 +22,9 @@
  */
 final class SecurityControllerBuilder
 {
-    private $phpCompatUtil;
-
-    public function __construct(PhpCompatUtil $phpCompatUtil)
-    {
-        $this->phpCompatUtil = $phpCompatUtil;
+    public function __construct(
+        private PhpCompatUtil $phpCompatUtil
```

Comment:
We should always end with a `,`, it makes future diffs easier to read, also there is a PHP-CS-Fixer rule

Reply:
Good catch! thanks Oskar. Done in #1127